config/creatrs/imp.cfg

```diff
@@ -99,8 +99,6 @@ PrisonKind = NULL
 TortureKind = NULL
 ; Creature properties:
 ;   BLEEDS - the creature leaves blood when is hit, slapped, dying or being injured in any way.
-;   UNAFFECTED_BY_WIND - the creature isn't pushed back by Wind spell.
-;   IMMUNE_TO_GAS - the creature isn't injured by Gas Traps and Farts.
 ;   HUMANOID_SKELETON - the creature leaves a skeleton if left in prison to die.
```

Comment:
If these are depreciated but supported, it could be worth keeping them and saying "(depreciated, see new Immunities section)"

Reply:
Yes, remove them from the documentation and the use in all bundled content. This is how I usually handle deprecated stuff.

Reply:
In this case, are there even any bundled maps that use them?